ensime_shared/client.py

```diff
@@ -224,26 +202,10 @@ def tell_module_missing(self, name):
         msg = feedback["module_missing"]
         self.editor.raw_message(msg.format(name, name))
 
-    # TODO: this should be the Ensime class's responsibility, not EnsimeClient;
-    #       move (and eliminate self.vim) after fixing, see #294
-    def disable_plugin(self):
-        """Disable plugin temporarily, including also related plugins."""
-        self.log.debug('disable_plugin: in')
-
-        def threadsafe_vim(command):
-            """Threadsafe call if neovim, normal if vim."""
-            def normal_vim(e):
-                self.vim.command(command)
-            with catch(Exception, normal_vim):
-                self.vim.session.threadsafe_call(command)
-
-        for path in self.runtime_paths:
-            self.log.debug(path)
-            threadsafe_vim('set runtimepath-={}'.format(path))
-
+    def _display_ws_warning(self):
         warning = "A WS exception happened, 'ensime-vim' has been disabled. " +\
             "For more information, have a look at the logs in `.ensime_cache`"
-        threadsafe_vim('echo "{}"'.format(warning))
+        self.editor.raw_message(warning)
```

Comment:
This string should probably go into `feedback` for sake of consistency with most other messaging. I'll save that for later.